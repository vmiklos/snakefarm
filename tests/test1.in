debug 1
out 1
time 100
load test1.map
addplayer 1
addsnake 1 1 0 1 1 0
2 1
endsnake
comment # itt a bemeneti fajlhoz hasonlo, vele azonos helyzetet eloallito parancsoknak kell szerepelnie, es letre kell jonnie egy test1_output.map-nek, test1.map-pal azonos terkepet generalo tartalommal
snapshot test1_output.map
exit
