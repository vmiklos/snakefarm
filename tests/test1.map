1;W;0;0
2;W;0;1
3;W;0;2
4;W;0;3

5;W;1;0
6;0;2;5
7;F;3;6
8;W;4;7

9;W;5;0
10;A;6;9
11;T;7;10
12;W;8;11

13;W;9;0
14;W;10;13
15;W;11;14
16;W;12;15