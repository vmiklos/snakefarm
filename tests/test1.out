Event MapLoad test1.map
# itt a bemeneti fajlhoz hasonlo, vele azonos helyzetet eloallito parancsoknak kell szerepelnie, es letre kell jonnie egy test1_output.map-nek, test1.map-pal azonos terkepet generalo tartalommal
load test1_output.map
addplayer 1
addsnake 1 1 0 1 1 0
2 1
endsnake
Event Exit
